modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -212,6 +212,46 @@ Password: ****
 After setting up a proper way to access Anypoint Platform from the CLI, you can start passing commands. +
 See xref:anypoint-platform-cli-commands.adoc[Anypoint Platform CLI 3.x List of Commands] for instructions on how to use them.
 
+=== Use Anypoint CLI Through a Network Proxy 
+
+To use Anypoint CLI through a proxy server, configure the `HTTP_PROXY` and `HTTPS_PROXY` environment variables with the hostname or IP addresses for your proxy servers.
+Use the following commands, depending on your operating system:
+
+* Linux, macOS, or Unix:
```

Comment:
```suggestion
* Linux, macOS, or Unix
```

Reply:
IMHO this colon is unnecessary because of the structure imposed by the list (as opposed to transferring from plain text to a display format).